### ISSUE TITLE
Can't register a new account to Taiga

### ISSUE THREAD

**Issue by cmoulliard** [fabric8io/fabric8-devops#76]
Version used : 2.2.43

![screenshot 2015-10-08 09 28 57](https://cloud.githubusercontent.com/assets/463790/10360183/1b37a244-6d9f-11e5-8717-2dad16002350.png)

**Comment by rawlingsj** [comment]
Yeah the taiga image was used for a PoC a little while ago, we've integrated with it but it needs a to be looked at again.  We've been concentrating on CD until now, over the next few weeks / months this will extend to the wider DevOps effort which includes Taiga.  

Once we've integrated with it a little better and addressed a couple of issues like the adding users or even better SSO we'll record a screencast showing how it fits together.

For now you should be able to log in with admin/123123.

**Comment by gobengo-travel** [comment]
I also just hit this issue. It's also pretty hard to debug what's going wronga s I think the relevant taiga-docker images are built in such a way that taiga's logging is disabled.

If I had to take a wild guess, the signup flow is trying to send an email for you to verify your email address, but the SMTP request is throwing an uncaught exception due to fake/example SMTP credentials in taiga.yml https://github.com/fabric8io/taiga-docker/blob/stable/taiga.yml#L23

**Comment by rawlingsj** [comment]
Ah that could well be it - yeah I think it's fair to say that folks aren't using this Taiga image in production.  I can try and take a look at this and see if we can avoid the SMTP altogether or even run a smtp server in the pod.  Unless you wanted to try doing that yourself?  I could help talk you through setting up a local development environment to test any changes to the image or pod?

**Comment by gobengo-travel** [comment]
@rawlingsj Thanks for the timely response. I found compelling fabric8's idea self-hosting all the pieces of software a dev team might need, and I arbitrarily chose taiga to try to install first (via helm on minikube).

I don't *really* need to get it running all the way, so I just figured I'd post here in case others have the same issue. If I need to set it up for a real-world project, I'll dig in and try to fix.

**Comment by rawlingsj** [comment]
Ok great - that sounds like a plan, thanks @gobengo-travel.  BTW there are some efforts that will provide an even better integrated developer experience coming soon.  Stay tuned :)
